### PR TITLE
Feat/some updated styling

### LINKED ISSUE #20: Upgrading CSS
Hi @kieranmillar 

I've really been enjoying playing through your sets on this site with my family. We'd been looking for some crafted sets rather than just random ones and these ones do the trick

I've been using it on my phone and I've noticed a few CSS tweaks that could improve the experience.

If you'd be interested I'd be happy to make some pull requests with some possible changes.

Some places I'm thinking:
* Larger/responsive Button sizes on smaller screens
* Larger/responsive select box sizes on smaller screen
* Larger/responsive tab sizes on smaller screens 

Let me know what you think, and I could have a look at it on saturday 😊

Thanks

### PR DESCRIPTION
**Pull request by squigglybob**
resolves #20

### CODE REVIEW

**Comment by squigglybob** [comment]
Hope you like it @kieranmillar  😃 

Everything that I was initially thinking of I've done.
I also added in a copy notice (which isn't technically just CSS updates)

I have some other thoughts if you are interested.

Let me know if there is anything you would like me to change

File: styles.css [hunk 1]
```diff
@@ -1,14 +1,81 @@
+/* Globals */
+
+* {
+	box-sizing: border-box;
+}
+
+html {
```
Comment (kieranmillar) [inline styles.css:7]: As body now has a maximum width, on screens larger than this, when the page scrolls vertically (such as switching to the FAQ tab) the content shifts to accommodate the new scrollbar that appeared.

I think the property `scrollbar-gutter: stable;` placed here in html class will sort this.

**Comment by kieranmillar** [comment]
Looks real good, I cloned it onto my computer and took a look and gave it a thorough test. Just one small comment.

Definitely interested in your other thoughts.

File: styles.css (hunk 1 above)
Comment (squigglybob) [reply styles.css:7]: 👍🏻 good spot. All done

**Comment by squigglybob** [comment]
Thanks for the comments.
I'll make a new issue with some thoughts for discussion 😄

**Review by kieranmillar — APPROVED** [review]

**Pull request merged**